testcases/OpTestLPM.py

```diff
@@ -121,14 +122,17 @@ def rmc_service_start(self, mg_system, remote_hmc=None):
             self.cv_HOST.host_run_command('/usr/sbin/rsct/install/bin/recfgct', timeout=120)
             self.cv_HOST.host_run_command('/opt/rsct/bin/rmcctrl -p', timeout=120)
             if not self.util.wait_for(self.is_RMCActive, timeout=300, args=[mg_system, remote_hmc]):
+                self.util.gather_os_logs(output_dirname="Test-Fail-Logs")
                 raise OpTestError("RMC connection is down!!")
 
     def lpm_failed_error(self, mg_system):
         if self.cv_HMC.is_lpar_in_managed_system(mg_system, self.cv_HMC.lpar_name):
             cmd = "lssyscfg -m %s -r lpar --filter lpar_names=%s -F state" % (
                    mg_system, self.cv_HMC.lpar_name)
             lpar_state = self.cv_HMC.ssh.run_command(cmd)[0]
+            self.util.gather_os_logs(output_dirname="Test-Fail-Logs")
             raise OpTestError("LPAR migration failed. LPAR is in %s state." % lpar_state)
+        self.util.gather_os_logs(output_dirname="Test-Fail-Logs")
```

Comment:
not very specific but some how   directory name not looks good , can this be just `TestFailLogs`